ametrine/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
This looks like an example, maybe put it under the examples instead of a free standing crate?

Reply:
I'm not sure if it can be an example, depends on whether `cargo-apk` supports that.